### PR TITLE
Bugfix 15865 Fixed description of is not among

### PR DESCRIPTION
**Pull request by bhall2001**
fixed description to correctly state that is not among implies string
does not exist.

### CODE REVIEW

File: docs/notes/bugfix-15865 [hunk 1]
```diff
@@ -0,0 +1 @@
+# Fixed Dictionary example for is within
```
Comment (livecodeali) [inline docs/notes/bugfix-15865]: @bhall2001 I think you've used the same bugfix note description as for the other PR!

Also, could you make sure the file has a `.md` suffix please?

**Review by livecodeali — CHANGES_REQUESTED** [review]

**Review by livecodeali — APPROVED** [review]

**Comment by livecodeali** [comment]
@livecode-vulcan review ok fb329e6

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok fb329e6a4a23913cd64b73b4e8f1f5fcca243549

**Comment by livecode-vulcan** [comment]
😎 test success fb329e6a4a23913cd64b73b4e8f1f5fcca243549
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/376)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/381)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/376)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/376)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/382)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/376)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/376)

**Pull request merged**